lib/ui/LandingPage.dart

```diff
@@ -7,11 +7,14 @@ import 'package:flutter/cupertino.dart';
 import 'package:flutter/material.dart';
 import 'package:intl/intl.dart';
 
-class _Strings {
-  static final englishAction = "English";
-  static final swahiliAction = "Swahili";
-  static final confirmAction = "Confirm";
-  static final cancelAction = "Cancel";
+class _LocalisedStrings {
+  static String englishAction() => Intl.message('English');
```

Comment:
probably don't want to translate the language names (as when switching you will only know your own language) eg. if you are English you might not know what English is, in Swahili in order to switch